### DISCUSSION TITLE
How to clone the repository correctly

### DISCUSSION THREAD

**Discussion by ning-drift** [Revolutionary-Games/Thrive#2185] [Q&A]
![Z)D @YLP`J~_BT(3`@FTV30](https://user-images.githubusercontent.com/71750287/113808023-b9eb6600-9797-11eb-9d01-194ef7d79385.png)
![open_error](https://user-images.githubusercontent.com/71750287/113808217-21a1b100-9798-11eb-8813-12f4c56b2e25.png)
Here is my clone log and the error message of opening the project for the first time.Did I not install the lfs extension correctly? Or what other reason, please teach me

**Comment by hhyyrylainen** [comment]
Just a guess, but using mingw might cause problems. I don't know if anyone has tried that on Windows.
Did you verify if the image files in the assets folder can be opened in an image viewer?
Did you try building the game from the Godot editor, if the build fails, you didn't setup either your build tools or nuget packages correctly.

**Reply by ning-drift** [reply]
Yes, the pictures of local assets\textures are all viewable. I use VS2017 to compile the project, the godot engine version is 3.2.3.stable.mono.official, and the VSMonoDebugger plugin is also installed.When I clicked the build button of the godot editor to compile, it`s fialed.the error code is error CS1617.Because of my Vs2017‘s language is Chinese, So the error log is also in Chinese.So I uploaded it as a reference
[error_log.txt](https://github.com/Revolutionary-Games/Thrive/files/6270350/error_log.txt)
It may be something wrong with my local environment, please help me

**Reply by hhyyrylainen** [reply]
Visual studio 2017 is too old, doesn't work.
This is the error that it doesn't support the latest C# version that Thrive requires:
```
CSC(0,0): error CS1617: /langversion 的选项“8.0”无效。使用 "/langversion:?" 列出支持的值。 [D:\godot\project\Thrive\Thrive.csproj]
```
